SCRAM/GMake/Makefile.rules

```diff
@@ -19,6 +19,16 @@ TARGETDIR:= $(subst /,_,$(THISDIR))
 .DEFAULT_GOAL := all
 .PHONY: FORCE_TARGET
 ##############################################################################
+ifeq ($(shell [ -t 1 ] > /proc/$${PPID}/fd/1 && echo terminal || echo pipe),terminal)
```

Comment:
@fwyzard , I think writing to `/proc/$${PPID}/fd/1` is causing the log file truncation